BUILDING.md

```diff
@@ -318,36 +318,44 @@ If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
 ```text
-$ tools/test.py test/parallel/test-stream2-transform.js
+$ <python-exec> tools/test.py test/parallel/test-stream2-transform.js
+```
+
+For example on Mac it will be:
+
+```text
+$ python3 tools/test.py child-process
```

Comment:
I think it's probably the same on Linux, maybe someone can confirm?

It would be great to add a windows example, or maybe it's the same alias?